### PR TITLE
Add support for delete_message_seconds ban argument 

### PR DESCRIPTION
**Pull request by bijij**
## Summary

Adds `delete_message_seconds` to `Guild/Member.ban` also deprecates `delete_message_days`.

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/guild.py [hunk 1]
```diff
@@ -3397,8 +3405,19 @@ async def ban(
...
             Banning failed.
+        TypeError
+            You specified both ``delete_message_days`` and ``delete_message_seconds``.
         """
-        await self._state.http.ban(user.id, self.id, delete_message_days, reason=reason)
+        if delete_message_days is not MISSING and delete_message_seconds is not MISSING:
+            raise TypeError('Cannot mix delete_message_days and delete_message_seconds keyword arguments.')
+
+        if delete_message_days is not MISSING:
+            delete_message_seconds = delete_message_days * 86400  # one day
```
Comment (bijij) [inline discord/guild.py:3415]: Would we want to raise a warning here? is there some way for us to do that currently, or would a new utility need to be made.

File: discord/guild.py [hunk 2]
```diff
@@ -3386,6 +3387,13 @@ async def ban(
         delete_message_days: :class:`int`
             The number of days worth of messages to delete from the user
             in the guild. The minimum is 0 and the maximum is 7.
+
+            .. deprecated:: 2.1
+        delete_message_seconds: :class:`int`:
+            The number of seconds woorth of messages to delete from the user
```
Comment (NCPlayz) [inline discord/guild.py:3393]:
```suggestion
            The number of seconds worth of messages to delete from the user
```

File: discord/guild.py (hunk 1 above)
Comment (Rapptz) [reply discord/guild.py:3415]: The mechanism for raising a warning here was pretty standard, I did something similar with `request_offline_members` in `Client.__init__`.

You can add `warnings.warn(msg, DeprecationWarning, stacklevel=2)`

Comment (bijij) [reply discord/guild.py]: Done.

**Pull request merged**